ops/terraform/services/common/s3-test-data.tf

```diff
@@ -0,0 +1,108 @@
+
+data "aws_canonical_user_id" "current" {}
+
+# bfd-public-test-data S3 bucket imports, not currently managed with terraform
+import {
```

Comment:
I wasn't even aware of this syntax. Seems straightforward enough, though there is the _potential_ this could break in the future:
> Experimental: While we do not expect to make backwards-incompatible changes to syntax, the -generate-config-out flag and how Terraform processes imports during the plan stage and generates configuration may change in future releases.